README.md

```diff
@@ -193,7 +193,7 @@ therefore, `-o false` does not work.
 
 ## `std::vector<T>` values
 
-Parsing of list of values in form of an `std::vector<T>` is also supported, as long as `T`
+Parsing of list of values in form of a `std::vector<T>` is also supported, as long as `T`
```

Comment:
Thanks for the fix, but I'm thinking that this is a terrible sentence and should probably be rewritten more to something like
> Parsing a list of values into a `std::vector<T>` is ...

Reply:
OK, I reworded that sentence some more. I also say "define" for a define symbol instead of "definition."